app/assets/stylesheets/blacklight_catalog/_static_pages.scss

```diff
@@ -20,31 +20,34 @@ p.static-blurb-body > a {
 }
 
 .content-row {
-  background: rgba(231,234,241, 0.5);
-/* Light Grey */
+  background: rgba(231, 234, 241, 0.5);
+  /* Light Grey */
 
-border: 1px solid #E7EAF1;
```

Comment:
@CB987 Can you add this color hex into the `_colors.scss` partial and reference by your new variable name?